### ISSUE TITLE
Courses progress 

### ISSUE THREAD

**Issue by shadston** [coders-school/site#81]
Hey, 

I have an UX improvement suggestion. It would be great to be able to see what is your progress on specific courses when you enter the 'Wszystkie moduly' page. I know there is something similar under profile settings in 'My moduly', but as far as I know not many people use that option.

**Comment by WojtekMs** [comment]
I agree with you I have just had the same idea about this improvement. Also I have to say that I didn't know about this feature in profile settings in 'Moje moduły', thanks!
